sig-auth/annual-report-2023.md

```diff
@@ -12,41 +12,62 @@
    - Governance and leadership changes
 -->
 
+   - Governance and leadership changes
+     - [**Mo Khan elected as new SIG tech lead**](https://groups.google.com/g/kubernetes-sig-auth/c/mHb4p8xWMR8/m/lk0UpMKXAAAJ).
+     - Previous SIG TL Mike Danese stepped down during 2023 and stayed on as a chair. Many thanks for his leadership and guidance over the years.
+   - The alpha `SecurityContextDeny` admission plugin was deprecated in [in v1.27](https://github.com/kubernetes/kubernetes/issues/111516) and removed in v1.30.
+     - The [Pod Security Admission](https://kubernetes.io/docs/concepts/security/pod-security-admission/) plugin enforcing the
+       [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) `Restricted` profile captures what this plugin was trying to achieve
+       in a better and up-to-date way.
+   - [KEP-3325: Review attributes of a current user](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/3325-self-subject-attributes-review-api) promoted to stable in v1.28.
+     - `whoami` kubectl command promoted from `kubectl alpha` to `kubectl` [in v1.27](https://github.com/kubernetes/kubernetes/pull/116510).
```

Comment:
```suggestion
     - `whoami` kubectl command promoted from `kubectl alpha` to `kubectl beta` [in v1.27](https://github.com/kubernetes/kubernetes/pull/116510).
```

*beta* missing?

Reply:
Otherwise this looks good to me.

Reply:
As per https://github.com/kubernetes/kubernetes/pull/116510, I only see the command being moved from `alpha` to `not-alpha`. IIUC there is no _beta_ stage for kubectl?